src/System.Runtime/tests/System/Attributes.cs

```diff
@@ -199,6 +199,13 @@ public static void NegTest()
             Assert.Throws<ArgumentNullException>(() => Attribute.GetCustomAttributes(typeof(TestClass2).GetMethod("TestMethod2"), null, false));
             Assert.Throws<ArgumentNullException>(() => Attribute.GetCustomAttributes(typeof(TestClass2).GetMethod("TestMethod2").GetParameters()[0], null, false));
         }
+
+        [Fact]
+        public static void MultipleAttributesTest()
+        {
+            Assert.Equal("System.Tests.MyCustomAttribute System.Tests.MyCustomAttribute", string.Join(" ", typeof(MultipleAttributes).GetCustomAttributes(inherit: false)));
```

Comment:
This is fine but note there exists an overload of Assert.Equal for IEnumerable<T>.